cmd/statusd/main.go

```diff
@@ -245,6 +246,11 @@ func makeNodeConfig() (*params.NodeConfig, error) {
 	nodeConfig.HTTPPort = *httpPort
 	nodeConfig.IPCEnabled = *ipcEnabled
 
+	if *ipcFile != "" {
```

Comment:
Minor issue: if I pass `-ipc=false` and `ipcfile=some/path.ipc`, the IPC will be enabled which is counter intuitive.